src/gui/qgsdatadefinedbutton.h

```diff
@@ -15,16 +15,30 @@
 #ifndef QGSDATADEFINEDBUTTON_H
 #define QGSDATADEFINEDBUTTON_H
 
-#include "qgsfield.h"
+#include <qgsfield.h>
+#include <qgsdatadefined.h>
+
+#include <QDialog>
 #include <QFlags>
 #include <QMap>
 #include <QPointer>
 #include <QToolButton>
+#include <QScopedPointer>
 
 class QgsVectorLayer;
 class QgsDataDefined;
 
 /** \ingroup gui
+ * \class Qgs
```

Comment:
All these added methods/classes need to be added to the SIP bindings too, and should have "added in ..." notes.